eva-tools/src/main/java/embl/ebi/variation/eva/vcfdump/VariantExporter.java

```diff
@@ -493,6 +189,7 @@ public int getFailedVariants() {
         Integer start = variant.getStart();
         Integer end = start + reference.length() -1;
         String filter = "PASS";
+        // TODO: avoid creating a list per variant
```

Comment:
The way the list is used in this method, an array could be initialized as `new String[] { reference, alternate }`, the iterators would work the same and the get/set calls could be replaced by direct index access.